src/components/FirstFilters.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './FirstFilters.module.css';
+
+const FirstFilters = () => {
+  return (
+    <article>
+      <div className={styles.bigButton}>
+        <Link to="/jeveuxtrouver" title="">
+          Je veux trouver THE film&nbsp;!
+        </Link>
+      </div>
+      <div className={styles.bigButton}>
+        <Link to="/filmChoice" title="Je partage le film que je veux regarder">
+          J’ai trouvé&nbsp;: je partage&nbsp;!
```

Comment:
Replace "trouvé" by "l'idée"

Reply:
Ok, I replaced it by "J'ai déjà l'idée : je partage !". Is this okay with you ?

Reply:
resolved on this commit : https://github.com/WildCodeSchool/remotefr-js-0920-p2-themoviedb/pull/1/commits/a3291347310956dcf5af255bd892e028cc04dfad